cmd/data-node/commands/dehistory/empty-datanode.go

```diff
@@ -0,0 +1,55 @@
+package dehistory
+
+import (
+	"fmt"
+
+	"code.vegaprotocol.io/vega/datanode/sqlstore"
+
+	"code.vegaprotocol.io/vega/cmd/vegawallet/commands/flags"
+	"code.vegaprotocol.io/vega/logging"
+	"code.vegaprotocol.io/vega/paths"
+
+	"code.vegaprotocol.io/vega/datanode/config"
+)
+
+type emptyDatanodeCmd struct {
+	config.VegaHomeFlag
+	config.Config
+}
+
+func (cmd *emptyDatanodeCmd) Execute(_ []string) error {
+	cfg := logging.NewDefaultConfig()
+	cfg.Custom.Zap.Level = logging.WarnLevel
+	cfg.Environment = "custom"
+	log := logging.NewLoggerFromConfig(
+		cfg,
+	)
+	defer log.AtExit()
+
+	vegaPaths := paths.New(cmd.VegaHome)
+	configFilePath, err := vegaPaths.CreateConfigPathFor(paths.DataNodeDefaultConfigFile)
+	if err != nil {
+		return fmt.Errorf("couldn't get path for %s: %w", paths.DataNodeDefaultConfigFile, err)
+	}
+
+	err = paths.ReadStructuredFile(configFilePath, &cmd.Config)
```

Comment:
As in https://github.com/vegaprotocol/vega/issues/7028 can we make sure `config.Config` can be overriden by CLI options?

Reply:
phils in the middle of that, some code I will need to pickup, gonna wait till he merges